app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/welcomescreen/WelcomeScreen.kt

```diff
@@ -0,0 +1,152 @@
+        )
+        Canvas(
+            modifier = Modifier.fillMaxSize(),
+        ) {
+            drawRect(
+                brush = brush,
+            )
+        }
+        Column(
+            modifier =
+                Modifier.align(alignment = Alignment.BottomCenter)
+                    .padding(horizontal = 28.dp, vertical = 42.dp),
+        ) {
+            val annotatedString =
+                buildAnnotatedString {
+                    withStyle(
+                        style = SpanStyle(color = welcomeScreenPrimaryTextColor),
+                    ) {
+                        append(stringResource(R.string.welcome_screen_text_part1))
+                    }
+                    withStyle(
+                        style =
+                            SpanStyle(
+                                color = welcomeScreenPrimaryTextColor,
+                                fontWeight = FontWeight.Bold,
+                            ),
+                    ) {
+                        append(stringResource(R.string.welcome_screen_text_part2))
+                    }
+                }
+            val buttonHorizontalGradientBrush =
+                Brush.horizontalGradient(
+                    colors = listOf(buttonFirstColor, buttonSecondColor),
+                )
+            Image(
+                painter = painterResource(R.drawable.img),
+                contentDescription = null,
+                contentScale = ContentScale.FillBounds,
+                modifier =
+                    Modifier
+                        .width(176.dp)
+                        .height(18.dp),
+            )
+            Spacer(
+                modifier = Modifier.height(20.dp),
+            )
+            Text(
+                text = annotatedString,
+                color = welcomeScreenPrimaryTextColor,
+                fontSize = 30.sp,
+                lineHeight = 36.sp,
+            )
+            Spacer(
+                modifier = Modifier.height(20.dp),
+            )
+            Text(
+                text = stringResource(R.string.welcome_screen_description),
+                color = welcomeScreenPrimaryTextColor,
+                fontSize = 16.sp,
+            )
+            Spacer(
+                modifier = Modifier.height(20.dp),
+            )
+            Button(
+                onClick = {},
+                modifier =
+                    Modifier.fillMaxWidth()
+                        .height(48.dp)
+                        .background(
+                            brush = buttonHorizontalGradientBrush,
+                            shape = RoundedCornerShape(size = 20.dp),
+                        ),
+                colors =
+                    ButtonDefaults.buttonColors(
+                        containerColor = Color.Transparent,
+                    ),
+            ) {
+                Text(
+                    text = stringResource(R.string.welcome_screen_button_text),
+                    fontSize = 20.sp,
```

Comment:
style here as well. on a typography you can use `.copy` property to pass the one-off values such as colors

Reply:
done